examples/redis-replication-group/main.tf

```
@@ -0,0 +1,68 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+provider "aws" {
+  region = "eu-west-2"
+  alias  = "replica"
+}
+
+#####
+# VPC and subnets
+#####
+
+data "aws_vpc" "main" {
+  default = true
+}
+
+data "aws_vpc" "replica" {
+  default = true
+
+  provider = aws.replica
+}
+
+data "aws_subnet_ids" "main" {
+  vpc_id = data.aws_vpc.main.id
+}
+
+data "aws_subnet_ids" "replica" {
+  vpc_id = data.aws_vpc.replica.id
+
+  provider = aws.replica
+}
+#####
+# Elasticache Redis
+#####
+
+module "redis_main" {
+  source = "../../"
+
+  name_prefix           = "redis-replication-group-example"
```

```suggestion
  name_prefix           = "redis-replication-example"
```

This will need to be shortened solely because of:
`expected length of replication_group_id to be in the range (1 - 40)`